### DISCUSSION TITLE
Different behavior regarding folding when opening a `.norg` file via Telescope

### DISCUSSION THREAD

**Discussion by xfzv** [nvim-neorg/neorg#713] [Q&A] [answered]
- When opening a `.norg` file from the shell with `nvim file.norg`, the file opens folded.
- When opening Neovim and then opening `file.norg` via Telescope, the file opens unfolded.

Video:

https://user-images.githubusercontent.com/78810647/212545850-ba0fa57d-33b6-458a-ab56-559418d3cf28.mp4

Is there a way to get the same behavior for both (the file opens folded)?

```console
❯ nvim -v
NVIM v0.8.2
Build type: Release
LuaJIT 2.1.0-beta
```

My Neorg config:

```lua
return {
	"nvim-neorg/neorg",
	build = ":Neorg sync-parsers",
	dependencies = "nvim-lua/plenary.nvim",
	config = function()
		require("neorg").setup({
			load = {
				["core.defaults"] = {},
				["core.clipboard"] = {},
				["core.clipboard.code-blocks"] = {},
				["core.norg.completion"] = {
					config = { engine = "nvim-cmp" },
				},
				["core.norg.concealer"] = {
					config = {
						dim_code_blocks = {
							enabled = true,
							conceal = true,
							content_only = true,
							-- Will adapt based on the `conceallevel` option.
							-- If `conceallevel` > 0, then only the content will be dimmed,
							-- else the whole code block will be dimmed.
							adaptive = true,
						},
						icon_preset = "diamond",
					},
				},
			},
		})
	end,
}
```

My Telescope config:

```lua
return {
	"nvim-telescope/telescope.nvim",
	dependencies = "nvim-lua/plenary.nvim",
	config = function()
		require("telescope").setup({
			defaults = {
				prompt_prefix = "🔍 ",
				selection_caret = " ",
				vimgrep_arguments = {
					"rg",
					"--color=auto",
					"--column",
					"--hidden",
					"--line-number",
					"--no-heading",
					"--smart-case",
					"--with-filename",
				},
			},
		})
	end,
}
```

Thanks!

**Comment by jghauser** [comment, marked as answer]
Once upon a time I opened [this issue with telescope](https://github.com/nvim-telescope/telescope.nvim/issues/559), and maybe that's the problem you're seeing? Although if you're only seeing it with `.norg` files then maybe not. Just thought I'd mention it here (also because the workarounds suggested towards the end work quite well!).

**Reply by xfzv** [reply]
Thanks! https://github.com/nvim-telescope/telescope.nvim/issues/559#issuecomment-1195895807 seems to do the trick for me.
